### ISSUE TITLE
how to use embeding layer in this case?

### ISSUE THREAD

**Issue by R1ncy** [dmlc/mxnet#4241]
Hi, I have some questions about the `mx.sym.Embedding`.
I have two document sets written in English and Japanese and I want to feed them to a 2 inputs net.

The net is like this. After the reshape layer, the shape is `(batch_size, 2, sentence_size, num_embed)`.
```
    en_input                   jp_input
       |                        |
    embed1                    embed2
       |________________________|
                  |
       concat(axis=1)&reshape
                .......
```
Asumming the vocab size is `9000`, English words are mapping to `[1,5000]`,
Japanese words are mapping to `[5001,9000]` and both of them use `0` for `padding`.

I'm not sure about it because the embedding api said `All the input values should be integers in the range [0, input_dim)`.

1. Can I directly set the `input_dim=5000` for `embed1` and `input_dim=4000` for `embed2`, if I want to use these two embedding layers?  What will happen if the input values of embed2 > the value of input_dim in this case?
Or should I build a new vocab for Japanese and map the word to `[1,4000]` and 0 for padding?
If they're wrong, should I use only `one embeding layer(input_dim=9000)` which contains words in 2 languages?

2. How two get the word vectors from a pre-trained model? 
Does the row index of the embed_weight correspond to the word index?
(e.g. the first row of the embed_weight is the word embedding of the word which is mapped to 1?)

Thanks a lot.
